features/src/main/feature/camel-features.xml

```diff
@@ -2044,6 +2044,13 @@
         <feature version='${project.version}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-webhook/${project.version}</bundle>
     </feature>
+    <feature name='camel-whatsapp' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-webhook</feature>
+        <feature>jackson</feature>
```

Comment:
jackson feature contains 2 more dependencies than required
`        <bundle dependency='true'>mvn:com.fasterxml.jackson.jr/jackson-jr-objects/${jackson2-version}</bundle>
        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-cbor/${jackson2-version}</bundle>`
is that ok, or should we define a separate feature for the 3 needed ones?

Reply:
I agree with you those 2 bundles should not be in the Jackson feature. I think, we need to remove them in the Jackson feature, and add them where it is really needed. Let me fix that.

Reply:
good catch, thanks

Reply:
for now, I included only the necessary bundles

Reply:
No keep it as it was, your feature was good only the Jackson feature had to be fixed

Reply:
I'm fixing it

Reply:
it's fixed, you can pull the changes

Reply:
Done